packages/flutter/lib/src/gestures/long_press.dart

```diff
@@ -247,6 +247,14 @@ class LongPressGestureRecognizer extends PrimaryPointerGestureRecognizer {
   /// The [duration] argument can be used to overwrite the default duration
   /// after which the long press will be recognized.
   ///
+  /// The [allowedButtonsFilter] argument only gives this recognizer the
```

Comment:
Nit: I think you can make this a template and reuse it in the `TapGestureRecognizer`.